apps/README.md

```diff
@@ -2,11 +2,17 @@
 
 This directory contains the apps we use for testing Expo. **Do not add new apps to this repository.** Each app increases our maintenance costs and complexity of different packages in `node_modules` that interact with each other in subtle ways. To keep this repository healthy and maintainable, we need to minimize the number of apps we have.
 
-- Test Suite: end-to-end tests that run on Expo
-- Native Component List: a showcase of the components and APIs in the default Expo preset
-- Home (not in this directory): the UI for the development client
-- Sandbox: a project excluded from Git where you can try out things locally
-- jest-expo Mock Generator: an app that uses reflection to generate Jest mocks for native modules. Eventually we can remove the need for this once we have a JSON schema for each module definition (Turbomodules) and generate the Jest mock from the JSON schema.
-- bare-expo: a project that uses the bare workflow and can load the Test Suite and Native Component List's JS.
+- #### [bare-expo](https://github.com/expo/expo/tree/master/apps/bare-expo)
+  A project that uses the bare workflow and can load the Test Suite and Native Component List's JS.
+- #### [jest-expo Mock Generator](https://github.com/expo/expo/tree/master/apps/jest-expo-mock-generator)
+  An app that uses reflection to generate Jest mocks for native modules. Eventually we can remove the need for this once we have a JSON schema for each module definition (Turbomodules) and generate the Jest mock from the JSON schema.
+- #### [Home](https://github.com/expo/expo/tree/master/home) (not in this directory)
+  The UI for the development client.
+- #### [Native Component List](https://github.com/expo/expo/tree/master/apps/native-component-list)
+  A showcase of the components and APIs in the default Expo preset.
+- #### [Sandbox](https://github.com/expo/expo/tree/master/apps/sandbox)
+  A project excluded from Git where you can try out things locally.
+- #### [Test Suite](https://github.com/expo/expo/tree/master/apps/test-suite)
+  End-to-end tests that run on Expo.
```

Comment:
I would rather suggest using [relative links](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files)
```suggestion
- #### [bare-expo](apps/bare-expo)
  A project that uses the bare workflow and can load the Test Suite and Native Component List's JS.
- #### [jest-expo Mock Generator](apps/jest-expo-mock-generator)
  An app that uses reflection to generate Jest mocks for native modules. Eventually we can remove the need for this once we have a JSON schema for each module definition (Turbomodules) and generate the Jest mock from the JSON schema.
- #### [Home](home) (not in this directory)
  The UI for the development client.
- #### [Native Component List](apps/native-component-list)
  A showcase of the components and APIs in the default Expo preset.
- #### [Sandbox](apps/sandbox)
  A project excluded from Git where you can try out things locally.
- #### [Test Suite](apps/test-suite)
  End-to-end tests that run on Expo.
```